web/client/components/widgets/enhancers/dependenciesToFilter.js

```diff
@@ -29,6 +29,7 @@ module.exports = compose(
             mapSync !== nextProps.mapSync
             || dependencies.viewport !== (nextProps.dependencies && nextProps.dependencies.viewport)
             || geomProp !== nextProps.geomProp
+            || (nextProps.layer && layer.loadingError !== nextProps.layer.loadingError)
```

Comment:
Can you add a test for mapSync change on loadingError change ?

Reply:
Will do.

Reply:
Wait, I'm not sure what you mean. mapSync change? Why would mapSync change?

Reply:
Sorry I meant dependenciesToFilter